src/psyclone/tests/psyGen_test.py

```diff
@@ -569,6 +569,19 @@ def test_sched_view(capsys):
     assert colored("Schedule", SCHEDULE_COLOUR_MAP["Schedule"]) in output
 
 
+def test_sched_getitem():
+    '''Test that Schedule have the [int] operators overloaded to return the
```

Comment:
Picky but s/have/has/ and s/operators/operator/

Reply:
Done